backend/src/api/group/content-types/group/schema.json

```diff
@@ -49,6 +49,20 @@
           "localized": true
         }
       }
+    },
+    "Type": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "enumeration",
+      "enum": [
```

Comment:
I did not know about enum, so I research about it. Could you explain why we need to use enum instead of using just text?

Reply:
I use enum when I know which values will be there for the user to choose and when there is usage of the enum. The usage could be getting all rows with this enum etc.

For example, first name won't be a got fit for enum as you cannot expect first name to be certain value. In this case text would be better.

For example if it is State of your address, you know that there will be these States Imagine you want to use that information and the type of State is TEXT. Then some people will write it as 'NY' some goes like 'New York' some goes like 'nyc' some goes 'newyork'. It will be very hard to get all people living in NY. In this case enum would be better.

Therefore as we specified what types will be there for the Groups, we are using enum to specify those.

Enums are just numbers like 1,2,3,4 etc. As it is hard to remember the values if those values are in numbers, so we are using text as representative to use instead of the numbers. 

Let me know if this explanation is sufficient and if you have any further questions.